windows.md

```diff
@@ -0,0 +1,29 @@
+Windows Node Support
+
+Windows nodes are partially supported by kubecost as of v1.93.0. Additional support is coming soon!
+* Deployment
+  * The cluster must have at least 1 linux node for the kubecost cost-model to run on
```

Comment:
just curious, why do we have this requirement? do we have plans to eliminate?

Reply:
Our docker builds for ARM/AMD... it may be possible to build for windows but this would require some testing? https://adamtheautomator.com/docker-build-tag/